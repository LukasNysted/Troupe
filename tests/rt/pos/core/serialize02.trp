datatype Atoms = TEST

let val rawobj = 
             ()
  
    val filename = "serialization_test"
    val _ = save (filename, rawobj)    
    val rawobj2 = restore (filename)
in printWithLabels (rawobj2)
;  printWithLabels (rawobj = rawobj2)
  
end 