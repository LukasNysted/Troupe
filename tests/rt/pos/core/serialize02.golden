2020-06-06T16:57:49.150Z [RTM] [32minfo[39m: Skipping network creation. Observe that all external IO operations will yield a runtime error.
()@{}%{}
true@{}%{}
>>> Main thread finished with value: ()@{}%{}
